sim/resources/stompymicro/robot_fixed.urdf

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
```

Comment:
This file should be created based on joints.py using `sim/scripts/create_fixed_torso.py`
I guess that is missing in the readme.

Reply:
Understood:
- Update URDF Velocity and Effort to that of joints.py

Reply:
should add the docs tbh

Reply:
it's in the readme